src/kinetics/include/antioch/troe_falloff.h

```diff
@@ -242,7 +236,7 @@ namespace Antioch
   {
      
     // Fcent = (1.-alpha)*exp(-T/T***) + alpha * exp(-T/T*) + exp(-T**/T)
-    StateType Fc = (1. - _alpha) * ant_exp(-T/_T3) + _alpha * ant_exp(-T/_T1);
+    StateType Fc = (1.L - _alpha) * ant_exp(-T/_T3) + _alpha * ant_exp(-T/_T1);
```

Comment:
Why did we switch from 1. to 1.L (which forces conversions to long double) rather than 1 (which would still be accurately treated as CoeffType and then StateType) here?